druid/src/scroll_component.rs

```diff
@@ -244,7 +244,7 @@ impl ScrollComponent {
         let usable_space = viewport_major - major_padding;
 
         let length = (percent_visible * viewport_major).ceil();
-        let length = length.max(bar_min_size).min(usable_space);
+        let length = length.clamp(bar_min_size, usable_space);
```

Comment:
This seems like a potential panic. Usable space could be below the minimum bar size.

Reply:
I complained about this upstream, and it sounded like they were going to change the lint. Dunno if it made it into 1.66 though.

Reply:
The stable clippy still complains about this the same way. It has that *note: clamp will panic if max < min, min.is_nan(), or max.is_nan()* - which is something at least.

Feels like an overeager lint to me. Now that it has reached stable maybe some people will step on this rake and complain.